### PR TITLE
QE-7182 | fix typos for endpoints and set duration to 60s

### PR DESCRIPTION
**Pull request by amalhotragannett**

### CODE REVIEW

File: elections-2020/src/artillery-utils.js [hunk 1]
```diff
@@ -55,37 +55,37 @@ module.exports = artilleryUtils = {
...
           name: `state-house-${market.name}`,
-          url: '/elections/results/fragmentPackages/resultsStateHouseSenate?' +
+          url: '/thorium/elections/results/fragmentPackages/resultsStateHouseSenate?' +
             `format=uwResponse&level=lower&site-code=${market.siteCode}&electionDate=2020-11-03&state=california`
         },
         {
           name: `single-race-${market.name}`,
-          url: '/elections/results/fragmentPackages/resultsSingleRace?' +
+          url: '/thorium/elections/results/fragmentPackages/resultsSingleRace?' +
             `format=uwResponse&site-code=${market.siteCode}&dataID=2018-11-06-senate-AZ-4021`
```
Comment (mmillgate) [inline elections-2020/src/artillery-utils.js:89]: why is the date on this one 2018-11-06?

File: elections-2020/src/artillery-utils.js [hunk 2]
```diff
@@ -55,37 +55,37 @@ module.exports = artilleryUtils = {
         },
         {
           name: `date-index-${market.name}`,
-          url: '/elections/results/fragmentPackages/resultsDateIndex?' +
+          url: '/thorium/elections/results/fragmentPackages/resultsDateIndex?' +
             `format=uwResponse&site-code=${market.siteCode}&electionDate=2020-11-03`
```
Comment (mmillgate) [inline elections-2020/src/artillery-utils.js:59]: I'm concerned with the hardcoded date values being set for a future date during the up coming AP test windows, does this need to be configurable to work within the AP testing windows, or do they accept future dates and provide valid data?

File: elections-2020/src/artillery-utils.js [hunk 3]
```diff
@@ -55,37 +55,37 @@ module.exports = artilleryUtils = {
...
           name: `national-${market.name}`,
-          url: '/elections/results/fragmentPackages/resultsNational?' +
+          url: '/thorium/elections/results/fragmentPackages/resultsNational?' +
             `format=uwResponse&site-code=${market.siteCode}&electionDate=2020-11-03&office=senate`
         },
         {
           name: `state-${market.name}`,
-          url: '/elections/results/fragmentPackages/resultsState?' +
+          url: '/thorium/elections/results/fragmentPackages/resultsState?' +
             `format=uwResponse&site-code=${market.siteCode}&electionDate=2020-11-03&state=california`
```
Comment (mmillgate) [inline elections-2020/src/artillery-utils.js:69]: if the state-market.name value is changing dynamically, should the `&state=california` value of the URL also be changing dynamically?  Does it make sense to be seeing/getting California data if the state-market.name was set to Utah as an example?

File: elections-2020/src/artillery-utils.js (hunk 1 above)
Comment (mmillgate) [reply elections-2020/src/artillery-utils.js:89]: should the `senate-AZ-4021` dataID QSP value be dynamically set as well based on single-race-market.name?

File: elections-2020/src/artillery-utils.js [hunk 4]
```diff
@@ -55,37 +55,37 @@ module.exports = artilleryUtils = {
...
           name: `us-house-single-state-${market.name}`,
-          url: '/elections/results/fragmentPackages/resultsStateUSHouse?' +
+          url: '/thorium/elections/results/fragmentPackages/resultsStateUSHouse?' +
             `format=uwResponse&site-code=${market.siteCode}&electionDate=2020-11-03&state=california`
         },
         {
           name: `state-senate-${market.name}`,
-          url: '/elections/results/fragmentPackages/resultsStateHouseSenate?' +
+          url: '/thorium/elections/results/fragmentPackages/resultsStateHouseSenate?' +
             `format=uwResponse&level=upper&site-code=${market.siteCode}&electionDate=2020-11-03&state=california`
```
Comment (mmillgate) [inline elections-2020/src/artillery-utils.js:79]: this url and the url for state-house below are basically the same endpoint the only difference is the level QSP value.   Is the performance based on the endpoint of the QSP?

File: elections-2020/src/artillery-utils.js [hunk 5]
```diff
@@ -55,37 +55,37 @@ module.exports = artilleryUtils = {
...
         },
         {
           name: `state-house-${market.name}`,
-          url: '/elections/results/fragmentPackages/resultsStateHouseSenate?' +
+          url: '/thorium/elections/results/fragmentPackages/resultsStateHouseSenate?' +
             `format=uwResponse&level=lower&site-code=${market.siteCode}&electionDate=2020-11-03&state=california`
```
Comment (mmillgate) [inline elections-2020/src/artillery-utils.js:84]: the format=uwResponse is acceptable for Tangent sites and they are not required for the home-page-summary, summary-bop, key-races flows?

**Review by mmillgate — CHANGES_REQUESTED** [review]
some clarification questions

File: elections-2020/src/artillery-utils.js (hunk 1 above)
Comment (amalhotragannett) [reply elections-2020/src/artillery-utils.js:89]: These were the URLS provided by Brendan that we had to run performance tests on. Yes, that is an option to set those dynamically. Let me find a way to optimize this if I can.

File: elections-2020/src/artillery-utils.js (hunk 2 above)
Comment (amalhotragannett) [reply elections-2020/src/artillery-utils.js:59]: Good point. URLs provided by Brendan. Will check for how election dates are working.

File: elections-2020/src/artillery-utils.js (hunk 4 above)
Comment (amalhotragannett) [reply elections-2020/src/artillery-utils.js:79]: The difference is the site code. I will inject state values for these.

File: elections-2020/src/artillery-utils.js (hunk 5 above)
Comment (amalhotragannett) [reply elections-2020/src/artillery-utils.js:84]: A very good catch. I will enable both `uwResponse` and `tangent`

File: elections-2020/src/artillery-utils.js (hunk 1 above)
Comment (amalhotragannett) [reply elections-2020/src/artillery-utils.js:89]: I tried setting this dynamic data but couldn't find a way in this short notice.

**Review by mmillgate — APPROVED** [review]

**Pull request merged**